README.rst

```diff
@@ -0,0 +1,134 @@
+in a model independent way. The program was designed to help a researcher 
+answer the question: "has my material undergone a phase transition between 
+these two measurements?".
+
+One approach is to compare the two PDFs in a plot and view the difference curve
+underneath. However, significant signal can be seen in the difference curve from
+benign effects such as thermal expansion (peak shifts) and increased thermal 
+motion (peak broadening) or a change in scale due to differences in incident flux,
+for example. PDFmorph will do its best to correct for these benign effects before 
+computing and plotting the difference curve. One measured PDF (typically the higher
+temperature one) is identified as the target PDF and the second PDF is then "morphed"
+by "stretching" (chaning the r-axis to simulate a uniform lattice expansion),
+"smearing" (broadening peaks through a uniform convolution to simulate increased
+thermal motion), and "scaling" (self-explanatory). PDFmorph will vary the amplitude
+of the morphing transformations to obtain the best fit between the morphed and the
+target PDFs, then plot them on top of each other with the difference plotted below.
+
+There are also a few other morphing transformations in the program.
+
+Finally, we note that PDFmorph should work on other spectra that are not PDFs,
+though it has not been extensively tested beyond the PDF.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, https://github.com/diffpy/diffpy.utils
+
+It is recommended that you use `Anaconda Python <https://www.anaconda.com/distribution/>`_ to conveniently install PDFmorph and its software dependencies 
+with a few concise commands. For example, on Ubuntu Linux some of the required
+software can be installed using ::
+
+	sudo apt-get install \
+	python-setuptools python-wxtools python-numpy \ 
+	python-matplotlib
+
+To install the remaining packages, see installation instructions at their
+respective web-pages.
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
+run. If you are unfamiliar with the CLI, consult online resources on how to get 
+started with it on your machine.
+
+If you're using Anaconda Python, you can install from the "conda-forge" 
+channel of Anaconda packages ::
+
+     conda config --add channels conda-forge
+     conda install diffpy.pdfmorph
+
+If you don't use Anaconda or prefer to install from sources, make sure the required
```

Comment:
I would leave this in the user manual documentation, but here just say "If you don't use Anaconda or prefer to install from sources, please see instructions in the installation section of the user manual documentation [here]"
[here] can point to the top level of the docs.  If you point to some specific subsection it is cute but can break easily.

Reply:
just to be clear, when you point to the detailed instructions in the manual, delete them from here.

Reply:
> when you point to the detailed instructions in the manual, delete them from here.

Can you clarify that part of your response?
Re: docs: I'll be sure to add the link/restructure it once the docs are up and running!

Reply:
This was an extension of the previous comment.....my suggestion is to not confuse people with the non-conda installation in the readme, but put it in the docs, delete it here, but point to it in the docs from here.

Reply:
Gotcha! Will show up in next commit